hellonode/server.js

```diff
@@ -15,11 +15,19 @@
  */
 
 var http = require('http');
+var os = require('os');
+
+process.on('SIGINT', function() {
+  console.log('shutting down...');
+  process.exit();
+});
 
 var handleRequest = function(request, response) {
   console.log('Received request for URL: ' + request.url);
   response.writeHead(200);
-  response.end('Hello, World!');
+  response.end('Hello, World!\nHostname: ' + os.hostname() + '\n');
 };
+
 var www = http.createServer(handleRequest);
+console.log('server listening on port 8080');
```

Comment:
you might want to put this message inside of a callback to the listen call, to ensure it doesn't print until the server is listening
```js
www.listen(port, ( ) => {
  console.log(`server is listening on port ${port}`);
});